doc/locale_api_WIP.md

```diff
@@ -0,0 +1,48 @@
+
+Note: ListFoo APIs combine the IDs and the translations. 
+If it's easier to return just the IDs and ask for the translations separately,
+let's use such different API.
+
+### Locales
+
+```
+ListLocales(
+  in  s     display_locale # "en_US.UTF-8"
```

Comment:
Do we need this param or should we use what @shundhammer mention on call, so always return label in form english - native e.g. `Czech - Čeština`

Reply:
OK, YaST does it the english+native way already